pkg/controller/controller_binding.go

```diff
@@ -87,12 +89,49 @@ func isServiceInstanceCredentialFailed(binding *v1alpha1.ServiceInstanceCredenti
 	return false
 }
 
+// setAndUpdateOrphanMitigation is for setting the OrphanMitigationInProgress
+// status to true, setting the proper condition statuses, and persisting the
+// changes via updateServiceInstanceCredentialStatus.
+func (c *controller) setAndUpdateOrphanMitigation(binding *v1alpha1.ServiceInstanceCredential, toUpdate *v1alpha1.ServiceInstanceCredential, instance *v1alpha1.ServiceInstance, serviceClass *v1alpha1.ServiceClass, brokerName string, errorStr string) error {
+	s := fmt.Sprintf("Starting orphan mitgation for ServiceInstanceCredential \"%s/%s\" for ServiceInstance \"%s/%s\" of ServiceClass %q at ServiceBroker %q, %v",
+		binding.Name,
+		binding.Namespace,
+		instance.Namespace,
+		instance.Name,
+		serviceClass.Name,
+		brokerName,
+		errorStr,
+	)
+	toUpdate.Status.OrphanMitigationInProgress = true
+	toUpdate.Status.OperationStartTime = nil
+	glog.V(5).Info(s)
+
+	c.setServiceInstanceCredentialCondition(
+		toUpdate,
+		v1alpha1.ServiceInstanceCredentialConditionReady,
+		v1alpha1.ConditionFalse,
+		errorServiceInstanceCredentialOrphanMitigation,
+		s,
+	)
+	c.setServiceInstanceCredentialCondition(
```

Comment:
We don't want to set the reason and message of the Failed condition to this. We want the reason and message to reflect why the binding is in a failed state. The **reason** that the binding is failed is not that orphan mitigation has started. Orphan mitigation started as a **consequence** of the binding failing.